grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -968,4 +970,16 @@ class OrderController {
         }
         render (status: 200, text: "Successfully imported template")
     }
+
+    def changeOrderItemStatus = {
```

Comment:
I feel like i'd like this to be splitted. One action for cancellingOrderItem and one action to restoringOrderItem